lib/server/services/qualityControlFlag/QcFlagTypeService.js

```diff
@@ -54,6 +60,50 @@ class QcFlagTypeService {
         return qcFlag;
     }
 
+    /**
+     * Create new QC Flag Type
+     * @param {object} [parameters] qc flag parameters
+     * @param {string} [parameters.name] qc flag name
+     * @param {string} [parameters.method] qc flag method
+     * @param {boolean} [parameters.bad] qc flag bad attribute
+     * @param {string} [parameters.color] qc flag color
+     * @param {number} [parameters.userId] qc flag creator id
+     * @param {number} [parameters.externalUserId] qc flag creator external id
```

Comment:
Isn't this an entity?

Reply:
not exactly, user related params has different names

Reply:
Why is that? I think it would be simpler to keep using entities, no?

Reply:
extracted user params and use Partial<QcFlagType>